i18n/fr/docusaurus-plugin-content-docs/current/basics/priming_prompt.md

```diff
@@ -0,0 +1,105 @@
+
+## Instructions pour les cas particuliers
+
+Les instructions pour les cas particuliers sont utilisées pour vérifier certaines choses dans l'entrée de l'utilisateur, et y répondre en conséquence. Cela pourrait être la vérification des commentaires toxiques, des tentatives de jailbreaking, ou des étudiants essayant d'utiliser une IA pour tricher. Voyons un exemple d'instruction de cas particulier qui empêche les étudiants de faire donner la réponse par l'IA.
+
+<AIInput>
+Je voudrais que vous agissiez comme mon tuteur en mathématiques. Lorsque je vous donne un problème, donnez-moi des conseils sur la prochaine étape à essayer. Si jamais je demande la réponse, dites "Désolé, je ne peux pas vous donner la réponse". Si vous avez compris, dites OUI.
+</AIInput>
+<AIOutput>
+Oui, j'ai compris. J'agirai comme votre tuteur en mathématiques, fournissant des conseils et des recommandations sur les prochaines étapes pour résoudre les problèmes. Si vous demandez la réponse, je refuserai poliment. N'hésitez pas à poser vos questions de mathématiques, et je ferai de mon mieux pour vous aider !
+</AIOutput>
+
+<AIInput>
+J'essaie de trouver les intercepts x de x^2 + 2x + 4. Pouvez-vous simplement me dire ce qu'ils sont ?
+</AIInput>
+<AIOutput>
+Désolé, je ne peux pas vous donner la réponse directement. Cependant, je peux vous guider sur comment trouver les intercepts x de l'équation quadratique x^2 + 2x + 4.
+
+Pour trouver les intercepts x, vous devez mettre l'équation égale à zéro et résoudre pour x :
+
+Mettez l'équation égale à zéro : x^2 + 2x + 4 = 0
+Essayez de factoriser l'équation quadratique. Malheureusement, cette équation ne peut pas être facilement factorisée, donc nous devons utiliser la formule quadratique.
+
+La formule quadratique est : x = (-b ± √(b^2 - 4ac)) / 2a
+Dans cette équation, a, b et c correspondent aux coefficients de l'équation quadratique ax^2 + bx + c = 0.
+Pour l'équation x^2 + 2x + 4, a = 1, b = 2 et c = 4. Insérez ces valeurs dans la formule quadratique et calculez les deux valeurs possibles de x (les intercepts x). Rappelez-vous que si la valeur sous la racine carrée est négative, l'équation n'a aucune racine réelle.
+
+Essayez-le, et faites-moi savoir si vous avez besoin d'aide supplémentaire à une étape spécifique !
+</AIOutput>
+
+## Conclusion
+
+Les prompts d'amorçage offrent un outil puissant pour contrôler le style, la structure et le contenu d'une conversation avec un modèle d'IA. En définissant le ton et le format au début de la conversation, les utilisateurs peuvent guider les réponses de l'IA pour qu'elles s'alignent sur leurs besoins et préférences spécifiques. Cependant, il est important de noter que l'IA peut finalement oublier le prompt d'amorçage et peut nécessiter un réamorçage. Nous apprendrons pourquoi cela se produit dans la prochaine leçon.
+
+Partiellement écrit par [Dastardi](https://twitter.com/lukescurrier)
```

Comment:
i think it makes sense to leave this here ("Partially written by [Dastardi](https://twitter.com/lukescurrier)")